### PR TITLE
fping: Add package

### PR DESCRIPTION
**Pull request by nikil**
Maintainer: me
Compile tested: ar71xx, x86_64
Run tested: x86_64

Description: sends ICMP ECHO_REQUEST packets to network hosts

Signed-off-by: Nikil Mehta nikil.mehta@gmail.com

### CODE REVIEW

File: net/fping/Makefile [hunk 1]
```diff
@@ -0,0 +1,52 @@
+#
+# Copyright (C) 2006-2016 OpenWrt.org
```
Comment (diizzyy) [inline net/fping/Makefile]: Please don't use OpenWrt.org copyright unless you are under contract.
https://www.lede-project.org/faq.html#who_holds_the_copyright_of_lede_code

File: net/fping/Makefile [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=fping
+PKG_VERSION:=3.13
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=http://fping.org/dist/
+PKG_MD5SUM:=a39134baca6bc6ced56191cb51fb0dc2
```
Comment (diizzyy) [inline net/fping/Makefile]: Please use SHA256 checksums

**Comment by nikil** [comment]
Done - thanks for the comments.

**Comment by KazukiShimada** [comment]
Hi.
What does prevent this from merging?

Thanks in advance.

**Comment by diizzyy** [comment]
@nikil 
netdata is going to need this once the new version gets released, you can please bump this to 3.15?

**Comment by nikil** [comment]
Yeah, I mean I can bump the version, but can someone tell me what is going on with merging the 10 or so packages I submitted PRs for?  I'm not getting much feedback as to whether they are going to be accepted?  I'd like to get a general idea before I put too much more work in.

**Comment by diizzyy** [comment]
@thess @hnyman 
Any comments if this pull req gets up to date?

**Comment by diizzyy** [comment]
@nikil 
It's needed for at least another port that I'm interested in so go for it :-)

**Comment by hnyman** [comment]
Let's merge it as it is now, and you can then update it. (and you can see from buildbot faillogs, if it really compiles for all platforms, or if you need to do other changes.)

> but can someone tell me what is going on with merging the 10 or so packages I submitted PRs for? I'm not getting much feedback as to whether they are going to be accepted? I'd like to get a general idea before I put too much more work in.

Like @thess commented on https://github.com/openwrt/packages/pull/3571#issuecomment-274136975  and @yousong earlier in https://github.com/openwrt/packages/pull/3572#issuecomment-266354712 , the new package PRs may be ignored if they are seen as too niche things, or too far away from the general embedded device/ router functionality. 

If other users show interest for the package in the PR (in the PR discussion), it is more likely to get accepted.

**Comment by KazukiShimada** [comment]
Thanks to all.

**Pull request merged**